README.md

```diff
@@ -47,13 +47,13 @@ Options:
 * History page
 * Random page
 * Recent changes page; RSS, Atom and JSON feeds available
-* Hyphae can be deleted (while still preserving history)
+* Hyphae can be deleted while still preserving history
 * Hyphae can be renamed (recursive renaming of subhyphae is also supported)
 * Light on resources
 * Authorization with pre-set credentials, registration
 * Basic Gemini protocol support
 
 ## Contributing
-Help is always needed. We have a [tg chat](https://t.me/mycorrhizadev) where some development is coordinated. You can also sponsor on [boosty](https://boosty.to/bouncepaw). Feel free to open an issue or contact directly.
+Help is always needed. We have a [tg chat](https://t.me/mycorrhizadev) where some development is coordinated. You can also sponsor us on [boosty](https://boosty.to/bouncepaw). Feel free to open an issue or contact us directly.
```

Comment:
Fun fact: one doesn't sponsor *us* on Boosty, they sponsor bouncepaw, who is me.

Reply:
oh